articles/multifactor-authentication/administrator/push-supported-devices.md

```diff
@@ -0,0 +1,17 @@
+---
+description: How to enable and use push notifications for Guardian.
+---
+
+## Devices Supported for Push Notifications
```

Comment:
Most of the devices mentioned bellow don't support push notifications but they support the OTP code generation (PN is only supported on iOS and Android). You can of stated that but it might look a little bit confusing under this title I think.